frontend/pages/index.page.tsx

```diff
@@ -38,13 +38,19 @@ interface GridItemProps {
   sx?: SxProps;
 }
 
+interface FeaturedContentProps {
+  featuredcontentData: Exclude<ModuleUnion, Topic>[];
+}
```

Comment:
I think a prop name like `featuredModules` might be better here. I dislike the use of "data," since it doesn't seem to convey any helpful information. The same could be said about `todayinhistoryData`, maybe `todayModules` for that one? It's not great, but better than `todayinhistoryData` IMO.

Reply:
Resolved: featuredcontentData -> featuredModules and todayinhistoryData -> todayModules